imap_processing/mag/mag_cdf_attrs.py

```diff
@@ -0,0 +1,105 @@
+"""Shared attribute values for MAG CDF files."""
+from imap_processing.cdf.defaults import GlobalConstants
+from imap_processing.cdf.global_attrs import (
+    AttrBase,
+    GlobalDataLevelAttrs,
+    GlobalInstrumentAttrs,
+    ScienceAttrs,
+)
+from imap_processing.mag import __version__
+
+text = (
+    "The IMAP magnetometer (MAG) consists of a pair of identical magnetometers "
+    "which each measure the magnetic field in three directions in the vicinity of "
+    "the spacecraft. "
+    "MAG will contribute to our understanding of the acceleration and transport "
+    "of charged particles in the heliosphere. "
+    "MAG design and assembly is led by Imperial College, London. See "
+    "https://imap.princeton.edu/instruments/mag for more details."
```

Comment:
That page is frustrating to read with the jumping header :) Good job having the patience to grab that text!

Reply:
Lol I hate the website design of that page, it's so jerky